instances/metadata.toml

```
@@ -0,0 +1,679 @@
+source_model = "Sienna"
+case_name = "modified_RTS_GMLC_DA_sys_NetTransport_Horizon24_Day29"
+hash = "66258a0e8497e1e17a702b769ebc9c4dcb1ef285993d4f1f41d9dda59878c5cc"
+integer_features = ["unit-commitment"]
+time_steps = 24
+system_nodes = 73
+problem_type = "unit-commitment"
+
+[[instance]]
+name = "Sienna_modified_RTS_GMLC_DA_sys_NetTransport_Horizon24_Day314-868ad371df2ae99f1427c0d6f5d6af4f1c520d51224598e1dc085b0736df5955.mps.gz"
+source_model = "Sienna"
+case_name = "modified_RTS_GMLC_DA_sys_NetTransport_Horizon24_Day314"
+hash = "868ad371df2ae99f1427c0d6f5d6af4f1c520d51224598e1dc085b0736df5955"
+integer_features = ["unit-commitment"]
+time_steps = 24
+system_nodes = 73
+problem_type = "unit-commitment"
+
+[[instance]]
+name = "Sienna_modified_RTS_GMLC_DA_sys_NetTransport_Horizon24_Day332-b8b4fb5c2c4b0a5f18333028c8d9a7de9d11a5607bb8b660b73cd8a2f617d26a.mps.gz"
+source_model = "Sienna"
+case_name = "modified_RTS_GMLC_DA_sys_NetTransport_Horizon24_Day332"
+hash = "b8b4fb5c2c4b0a5f18333028c8d9a7de9d11a5607bb8b660b73cd8a2f617d26a"
+integer_features = ["unit-commitment"]
+time_steps = 24
+system_nodes = 73
+problem_type = "unit-commitment"
+
+[[instance]]
+name = "Sienna_modified_RTS_GMLC_DA_sys_NetTransport_Horizon48_Day29-baddda2ca9ed73b7a9935f88b30a217e31cfa48670e659a44d07142f65039029.mps.gz"
+source_model = "Sienna"
+case_name = "modified_RTS_GMLC_DA_sys_NetTransport_Horizon48_Day29"
+hash = "baddda2ca9ed73b7a9935f88b30a217e31cfa48670e659a44d07142f65039029"
+integer_features = ["unit-commitment"]
+time_steps = 48
+system_nodes = 73
+problem_type = "unit-commitment"
+
+[[instance]]
+name = "Sienna_modified_RTS_GMLC_DA_sys_NetTransport_Horizon48_Day314-ec745b9b237a474e1ed2a2ef455cba4c7cc926d59742ae1fd4930415cc1a46d4.mps.gz"
+source_model = "Sienna"
+case_name = "modified_RTS_GMLC_DA_sys_NetTransport_Horizon48_Day314"
+hash = "ec745b9b237a474e1ed2a2ef455cba4c7cc926d59742ae1fd4930415cc1a46d4"
+integer_features = ["unit-commitment"]
+time_steps = 48
+system_nodes = 73
+problem_type = "unit-commitment"
+
+[[instance]]
+name = "Sienna_modified_RTS_GMLC_DA_sys_NetTransport_Horizon48_Day332-c5f9c340dae227b689ac284a47d285373a0cd023658a17e9e56bc68136fad3de.mps.gz"
+source_model = "Sienna"
+case_name = "modified_RTS_GMLC_DA_sys_NetTransport_Horizon48_Day332"
+hash = "c5f9c340dae227b689ac284a47d285373a0cd023658a17e9e56bc68136fad3de"
+integer_features = ["unit-commitment"]
+time_steps = 48
+system_nodes = 73
+problem_type = "unit-commitment"
+
+[[instance]]
+name = "SpineOpt_1_electrolyzer_with_rolling_horizon.json-70fb7507bb16b4ea8e589a0890dc8bee3b8d3ca8acebede5e8592e693b22a646.mps.gz"
+source_model = "SpineOpt"
+case_name = "1_electrolyzer_with_rolling_horizon.json"
+hash = "70fb7507bb16b4ea8e589a0890dc8bee3b8d3ca8acebede5e8592e693b22a646"
+integer_features = ["other"] # review: reactor?
+time_steps = 24
+system_nodes = 1 # one "power" node
```

My proposal is to sum all the nodes, no matter the sector and define the integer feature as unit commitment since this comes from the operation of a chemical reactor operation. So, it is similar to a unit-commitment dispatch, with online/startup/shutdown variables).

```suggestion
integer_features = ["unit-commitment"]
time_steps = 24
system_nodes = 14
```

---

fine for me!